### PR TITLE
add stdev,argmin,argmax & set op types

### PR DESCRIPTION
**Pull request by schehata**
Few aggregation types were missing, this PR adds them:
- argmin
- argmax
- set
- variance
- countif
- distinctif

### CODE REVIEW

**Comment by lukasmalkmus** [comment]
Can you check `axiom-go` again? Looks like this is also missing `variance`, `distinctIf` and `countIf`.

**Comment by schehata** [comment]
> Can you check `axiom-go` again? Looks like this is also missing `variance`, `distinctIf` and `countIf`.
updated

**Review by lukasmalkmus — COMMENTED** [review]
Looks good but let's have this packed by @lilakurse as well.

File: lib/datasets.ts [hunk 1]
```diff
@@ -110,6 +110,13 @@ export namespace datasets {
         Topk = 'topk',
         Percentiles = 'percentiles',
         Histogram = 'histogram',
+        Variance = 'variance',
+        Stdev = 'stdev',
+        ArgMin = 'argmin',
+        ArgMax = 'argmax',
+        Set = 'set',
```
Comment (lilakurse) [inline lib/datasets.ts:117]: Maybe we should do MakeSet = "makeset" here

Comment (lukasmalkmus) [reply lib/datasets.ts:117]: This must match the server side! So what is it on the server side?

Comment (lukasmalkmus) [reply lib/datasets.ts:117]: After talking to @lilakurse we figured this has to be `makeset`.

Comment (schehata) [reply lib/datasets.ts]: updated

**Pull request merged**